src/vmm/src/vmm_config/logger.rs

```diff
@@ -120,13 +120,14 @@ pub fn init_logger(
         .set_include_origin(logger_cfg.show_log_origin, logger_cfg.show_log_origin)
         .set_include_level(logger_cfg.show_level);
 
+    let writer = FifoLineWriter::new(
```

Comment:
With this change, can we still use regular files with the logger? Our logger documentation says that you can use regular files as well and we just renamed `logger_fifo` to `logger_path` for that reason. Do we get any benefit from using `FifoLineWriter` instead of `Writer`?

Reply:
Yes, we can still use regular files. There is no change in behavior here. No benefit or penalty. Just renaming. And removin the `impl Write for Writer` which wasn't needed, since it was just forwarding the calls to the underlying `LineWriter`.

Maybe `open_fifo` is not necessarily the best name. Maybe `open_fd_nonblock` would be better. And `NonblockFdLineWriter`.

Reply:
Ah, okay, I see. So I thought that `open_fifo` and `FifoLineWriter` are functions from std that operated only with fifos. My bad. I just saw that you are defining them in `mod.rs`